pynance/data/retrieve.py

```diff
@@ -17,17 +17,19 @@
 
 import pandas_datareader.data as web
 
-def get(equity, start, end):
+def get(equity, *args, **kwargs):
     """ 
     Get DataFrame for an individual equity from Yahoo!  
     
+    .. versionchanged:: 0.5.0
+    
```

Comment:
that's a bad miss. sorry. i was so fixated on making it pass ..... sending it back